js/app/views/sounds.js

```diff
@@ -52,11 +54,32 @@ define("views/sounds", function(require) {
 
             this.render();
         },
+        filterCollectionByCid: function(cid) {
+            this.collection = this.data.collection.filterByCid(cid);
+
+            this.render();
+        },
         manageSounds: function(args) {
             this.stopPlayingSound();
 
             Radio.channel("Sounds").trigger("sound:play", args.model.getSlug());
         },
+        randomSound: function() {
+            this.stopPlayingSound();
+
+            this.filterCollection("");
```

Comment:
il y a `this.filterCollection` et `this.filterCollectionByCid` ?

Reply:
FilterCollection filtre sur le titre, par expression régulière, alors que ByCid ne filtre que par Id. Techniquement, filterByCid pourrait être renommé getByCid. J'ai pas d'imagination ou d'avis pour le coup, je t'écoute.

Reply:
ok, ça fera l'affaire 😉